src/AFQMC/HamiltonianOperations/Real3IndexFactorization_batched_v2.hpp

```diff
@@ -0,0 +1,629 @@
+//////////////////////////////////////////////////////////////////////
```

Comment:
Why is there _v2 version for these files. Multiple versions? Can we just include one?

Reply:
Possibly. The original version is faster, but uses about 30% more memory. The v2 version is probably the one to keep, due to memory limitations on gpu. With c++17, we can keep both versions on a single file.